docs/customization/backend.rst

```diff
@@ -4,9 +4,9 @@ Working with Python Code
 Managing Dependencies
 ---------------------
 
-To guarantee repeatable installations, all project dependencies are managed using `Pipenv <https://pipenv.readthedocs.io/en/latest/>`_.
-Project's direct dependencies are listed in ``Pipfile`` and running :code:`pipenv lock` would generate ``Pipfile.lock`` that has all versions pinned.
+To guarantee repeatable installations, all project dependencies are managed using `Poetry <https://poetry.eustace.io/docs/>`_.
+Project's direct dependencies are listed in ``pyproject.toml`` and running :code:`poetry lock` would generate ``poetry.lock`` that has all versions pinned.
 
-For users who are not using Pipenv, ``requirements.txt`` and ``requirements_dev.txt`` are also provided. They should be updated by :code:`pipenv lock -r > requirements.txt` and :code:`pipenv lock -r --dev > requirements_dev.txt`, if any dependencies are changed in ``Pipfile``.  
+For users who are not using Poetry, ``requirements.txt`` and ``requirements_dev.txt`` are also provided. They should be updated by :code:`poetry export --dev -f requirements.txt; mv requirements.txt requirements_dev.txt` and :code:`poetry export -f requirements.txt`, if any dependencies are added/updated. Additionally `--without-hashes` can be provided if hashes are not needed in requirement files.
```

Comment:
Why not ```poetry export --dev -f requirements_dev.txt``` instead of ```poetry export --dev -f requirements.txt; mv requirements.txt requirements_dev.txt```?

Reply:
Because `-f` is not `file` as one would normally think. It's `format` and there's only one format for now, which is `requirement.txt`. It always writes to a `requirements.txt` file, that's why I use mv for the dev dependencies.

Reply:
Got it.

Reply:
Alright, yes I just checked, the only recognized format is `requirements.txt`. It doesn't seem to be planned to allow us to do such things... It could be handy to have a `poetry export --dev -f requirements.txt -o requirements_dev.txt` command.

If someone needs the help page, below it is.
```
(venv) [sayuri@undefined venv]$ poetry export -h
USAGE
  poetry export [-f <...>] [--without-hashes] [--dev]

OPTIONS
  -f (--format)          Format to export to.
  --without-hashes       Exclude hashes from the exported file.
  --dev                  Include development dependencies.

GLOBAL OPTIONS
  -h (--help)            Display this help message
  -q (--quiet)           Do not output any message
  -v (--verbose)         Increase the verbosity of messages: "-v" for normal output, "-vv" for more verbose output and "-vvv" for debug
  -V (--version)         Display this application version
  --ansi                 Force ANSI output
  --no-ansi              Disable ANSI output
  -n (--no-interaction)  Do not ask any interactive question
```

Reply:
C.f. https://github.com/sdispater/poetry/issues/1015

Reply:
`-o <file>` has been merged, just need to wait for a release.